submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,17 @@
+### Complexity Corner: The Mall <author>The Kool</author>
+
+Welcome back once more to the Complexity Corner.  Today I'll be exploring a technique instead of a technology: the Mall.  As always with these articles, we'll be asking three questions: 'What is it?', 'What results can it give?', and 'Where should it be used?'
+
+What is this 'Mall' I keep hearing about?
+
+We've all experienced it by now, setting up something to craft manually and then drumming your fingers on the desk while you wait for the mile-long queue to finish.  Wouldn't it be so much nicer to walk over to a chest and have some waiting for you?  To just go shopping for your next project, instead of crafting by hand?  That's what the Mall is for.  Without even realizing it, you probably only think of automating things that have an automated end goal, such as ammo for your turrets or science for your labs.  The Mall is a place where you automate things that have a manual end goal - You.
+
+So what's the benefit?
+
+A typical early-game smelting line consists of 24 stone furnaces if you're planning ahead to upgrade them.  In order to feed these furnaces, you need 48 inserters, 24 long-handed inserters, and ~100 belts.  If you carry around only raw materials like plates and craft these things as needed, that's 12s for the furnaces, 50s for the belts, and 93s for the inserters; a total of 2m 35s that you're waiting around before you can build that one furnace line.  Multiply that by every furnace line, every mining outpost, and every factory cell you build and you'll soon find you're standing around waiting far more than you thought.  The mall makes that go away, building things for you while you build your factory.
+
```

Comment:
Another benefit you could talk about is having all this stuff ready when you unlock robots. I.e. The mall then becomes the production center for robot construction materials by just converting the outputs to passive provider chests.

Reply:
the remark is still valid for this section: it could benefit the evocation of bots boxes for mi/late game bot-assisted construction